vaultpress.php

```diff
@@ -651,6 +680,14 @@ function ui_delete_vp_settings_button() {
 		<?php
 	}
 
+	function ui_logo() {
+		?>
+		<div class="clearfix"></div>
+		<?php
+		$logo = new Automattic\Jetpack\Assets\Logo();
```

Comment:
Isn't it customary for a namespace reference to have a leading slash? Like `new \Automattic\Jetpack\Assets\Logo()`?

Reply:
https://stackoverflow.com/questions/4849279/importing-classes-and-namespaces-in-php-what-difference-does-a-leading-backslas

not sure what's the conclusion 😅

Reply:
AFAIK, leading backslashes aren't necessary. In addition to the post @lezama linked, here's some evidence from the PHP documentation to back that up:

> Note that for namespaced names (fully qualified namespace names containing namespace separator, such as Foo\Bar as opposed to global names that do not, such as FooBar), the leading backslash is unnecessary and not recommended, as import names must be fully qualified, and are not processed relative to the current namespace.

Source: https://php.net/manual/en/language.namespaces.importing.php#language.namespaces.importing